File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/low_level/outtake/SetLiftPositionCommand.java
package org.firstinspires.ftc.teamcode.commands.low_level.outtake;

import com.arcrobotics.ftclib.command.InstantCommand;
import org.firstinspires.ftc.teamcode.managers.OuttakeManager;

public class SetLiftPositionCommand extends InstantCommand {

    public SetLiftPositionCommand(OuttakeManager manager, OuttakeManager._LiftState state) {
        super(() -> manager.update(state));
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/low_level/outtake/SetLiftPositionCommand.kt
package org.firstinspires.ftc.teamcode.commands.low_level.outtake;

import com.arcrobotics.ftclib.command.InstantCommand;
import org.firstinspires.ftc.teamcode.managers.OuttakeManager;

class SetLiftPositionCommand(
        manager: OuttakeManager,
        state: OuttakeManager._LiftState,
        targetPosition: Int? = null
) : InstantCommand({
    manager.update(state, targetPosition ?: 0)
})